content/sql/concepts/dates/terms/between/between.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'BETWEEN()'
+Description: 'Returns records where a specified value is within a given range, including the boundary values.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Date'
+  - 'Queries'
+  - 'SQL Server'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The **`BETWEEN()`** operator is used to find values within a given range. The values can be numbers, texts, or dates.
+
+The **`BETWEEN()`** operator is _inclusive_. Both begin and end values are included.
+
+## Syntax
+
+```sql
+SELECT column_name(s) from table_name where column_name between value1 and value2;
+```
+
+> **Note** The basic syntax for `BETWEEN()` is generally same across most databases, but some differences may arise depending on how each database handles specific data types or expressions. You have to look for Data handling, Null handling, Case Sensitivity and Collations when using the operator on your database.
```

Comment:
Let's link 'data types' to its doc entry and do some minor changes:

```suggestion
> Note: The basic syntax for `BETWEEN` is generally same across most databases, but some differences may arise depending on how each database handles specific [data types](https://www.codecademy.com/resources/docs/sql/data-types) or expressions. To use this operator on a particular database, concepts such as _data handling_, _null handling_, _case sensitivity_, and _collations_ need to be checked first to avoid errors.
```